tests/framework/PostProcessors/InterfacedPostProcessor/test_interfacedPP.xml

```diff
@@ -80,10 +79,6 @@
     </MonteCarlo>
   </Samplers>
 
-  <Databases>
-    <HDF5 name="out_db" readMode="overwrite"/>
-  </Databases>
```

Comment:
Could you provide a reason why the database is removed?

Reply:
I removed since it was not part of the testing capability